### PR TITLE
[FR] Update `Article_styling_criteria\Formatting` and `Article_styling_criteria\Writing`

### PR DESCRIPTION
**Pull request by DoryanR**
Update of the translations of the following wiki pages:

- `Article_styling_criteria\Formatting`
- `Article_styling_criteria\Writing`
- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by Biokax — APPROVED** [review]

**Comment by TicClick** [comment]
please add changes from https://github.com/ppy/osu-wiki/pull/7022

File: wiki/Article_styling_criteria/Formatting/fr.md [hunk 1]
```diff
@@ -774,7 +808,12 @@ Voici un exemple de ce à quoi doit ressembler un tableau :
 
 ## Citations en bloc
 
-Les citations en bloc sont limitées à la citation du texte d'une personne. Elles ne doivent pas être utilisées pour formater le texte.
+Les citations en bloc sont limitées à la citation du texte d'une personne. Elles ne doivent pas être utilisées pour formater le texte. Le nom de la personne citée qui suit la citation doit être précédé d'un tiret (`-`) comme suit :
+
+```markdown
+> profitez du jeu
+—rrtyui
```
Comment (Walavouchey) [inline wiki/Article_styling_criteria/Formatting/fr.md:815]:
```suggestion
> profitez du jeu

-rrtyui
```

File: wiki/Article_styling_criteria/Formatting/fr.md [hunk 2]
```diff
@@ -277,23 +272,62 @@ Une note doit être placée à l'endroit approprié dans une section, mais doit
...
-Les noms des œuvres ou des jeux vidéo doivent être en italique. osu! (le jeu) est exempté de cette règle.
-
 La première occurrence d'une abréviation, d'un acronyme ou d'un sigle peut être mise en italique.
 
-L'italique peut également être utilisé pour accenter un ou plusieurs mots ou faciliter la lecture.
+L'italique peut également être utilisé pour accentuer un ou plusieurs mots ou faciliter la lecture.
+
+Les noms des œuvres ou des jeux vidéo doivent être en italique. osu! en est exempt.
+
+Par exemple, lorsque l'on fait référence à des musique sous le format `{artist} - {title}`, toute la partie est une référence à l'œuvre et doit donc être mise en italique :
```
Comment (Ganondorf-1) [inline wiki/Article_styling_criteria/Formatting/fr.md:304]:
```suggestion
Par exemple, lorsque l'on fait référence à des musiques sous le format `{artist} - {title}`, toute la partie est une référence à l'œuvre et doit donc être mise en italique :
```

File: wiki/Article_styling_criteria/Formatting/fr.md [hunk 3]
```diff
@@ -277,23 +272,62 @@ Une note doit être placée à l'endroit approprié dans une section, mais doit
...
+
+```markdown
+*cYsmix - triangles* est l'une des trois musique d'intro que l'on peut entendre au démarrage du jeu.
```
Comment (Ganondorf-1) [inline wiki/Article_styling_criteria/Formatting/fr.md:307]:
```suggestion
*cYsmix - triangles* est l'une des trois musiques d'intro que l'on peut entendre au démarrage du jeu.
```
```suggestion
*cYsmix - triangles* est l'une des trois musique d'intro que l'on peut entendre au démarrage du jeu.
```

Comment (Ganondorf-1) [inline wiki/Article_styling_criteria/Formatting/fr.md:307]:
```suggestion
*cYsmix - triangles* est l'une des trois musiques d'intro que l'on peut entendre au démarrage du jeu.
```

File: wiki/Article_styling_criteria/Formatting/fr.md [hunk 4]
```diff
@@ -277,23 +272,62 @@ Une note doit être placée à l'endroit approprié dans une section, mais doit
...
+
+Par ailleurs, les noms d'artistes ne sont généralement pas en italique. Cela signifie que dans les références en forme libre, seul le titre doit être mis en italique, car le nom de l'artiste ne fait alors pas partie du nom de l'œuvre :
+
+```markdown
+*Blue Zenith* de xi est une musique réputée dans la communauté d'osu! en raison d'un score célèbre réalisé par un joueur de haut niveau sur une certaine beatmap.
+```
+
+### Accentuation et liens
+
+Les liens apparaissent dans une couleur différente, ce qui le met déjà en valeur et ne nécessite donc pas de mise en valeur supplémentaire :
```
Comment (Ganondorf-1) [inline wiki/Article_styling_criteria/Formatting/fr.md:318]:
```suggestion
Les liens apparaissent dans une couleur différente, ce qui le met déjà en valeur et ne nécessitent donc pas de mise en valeur supplémentaire :
```

File: wiki/Article_styling_criteria/Formatting/fr.md [hunk 5]
```diff
@@ -277,23 +272,62 @@ Une note doit être placée à l'endroit approprié dans une section, mais doit
...
+
+```markdown
+[Camellia - OOPARTS](https://cametek.bandcamp.com/track/parts) est un exemple de musique officiellement créée spécifiquement pour osu!, autrement dit un *osu! original*, puisqu'elle a été spécifiquement commandée pour le concours osu! World Cup 2020 tiebreaker.
```
Comment (Ganondorf-1) [inline wiki/Article_styling_criteria/Formatting/fr.md:321]:
```suggestion
[Camellia - OOPARTS](https://cametek.bandcamp.com/track/parts) est un exemple de musique officiellement créée spécifiquement pour osu!, autrement dit un *osu! original*, puisqu'elle a été spécifiquement commandée pour le tiebreaker de l'osu! World Cup 2020.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Comment by Walavouchey** [comment]
@Ganondorf-1, @Biokax, I've applied the reviews, think this is good to go? The changes from #7022 can be added in a separate pr

**Review by Ganondorf-1 — APPROVED** [review]
Yes, I will make another PR as soon as possible

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**